packages/front-end/services/utils.tsx

```diff
@@ -53,3 +54,26 @@ export function isNullUndefinedOrEmpty(x) {
   if (typeof x === "object" && !Object.keys(x).length) return true;
   return false;
 }
+
+type EncryptedFeatureHash = {
+  content: string;
+  iv: string;
+};
+
+export const decryptFeatureList = (
+  hash: EncryptedFeatureHash,
+  secretKey: string
+) => {
+  const decipher = crypto.createDecipheriv(
+    "aes-256-ctr",
+    secretKey,
+    Buffer.from(hash.iv, "hex")
+  );
+
+  const decrpyted = Buffer.concat([
+    decipher.update(Buffer.from(hash.content, "hex")),
+    decipher.final(),
+  ]);
+
+  return decrpyted.toString();
+};
```

Comment:
This should live in the SDK docs, not here.  We don't need to decrypt anything in our app, we just need to tell people how to decrypt features from their own apps.  Also, try to avoid using the "crypto" library and instead use the standard SubtleCrypto methods if we can (https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/decrypt)